auth-tokens-filter/src/main/java/com/palantir/tokens2/auth/http/BearerTokenLoggingFilter.java

```diff
@@ -79,6 +79,6 @@ private void setUnverifiedContext(ContainerRequestContext requestContext, String
     }
 
     public static String getRequestPropertyKey(String key) {
-        return "com.palantir.tokens.auth." + key;
+        return "com.palantir.tokens2.auth." + key;
```

Comment:
http-remoting chose the approach of changing, so I decided to take the same approach